loopd/daemon.go

```diff
@@ -498,21 +499,53 @@ func (d *Daemon) initialize(withMacaroonService bool) error {
 		}
 	}
 
+	// Start the notification manager.
+	notificationCfg := &notifications.Config{
+		Client:    loop_swaprpc.NewSwapServerClient(swapClient.Conn),
+		FetchL402: d.impl.Server.FetchL402,
+	}
+	notificationManager := notifications.NewManager(notificationCfg)
+
+	d.wg.Add(1)
+	initChan := make(chan struct{})
+	go func() {
+		defer d.wg.Done()
+
+		log.Info("Starting notification manager")
+		err := notificationManager.Run(d.mainCtx, initChan)
+		if err != nil {
+			d.internalErrChan <- err
+			log.Errorf("Notification manager stopped: %v", err)
+		}
+	}()
+
+	select {
+	case <-time.After(10 * time.Second):
+		return fmt.Errorf("notification manager not ready")
```

Comment:
Is 10 seconds sufficient? If L402 fails and retries, `fetchL402` already waits for 10 seconds, so it will crash here by the time of the second attempt. Also in Tor 10 seconds is not enough.